app/services/ccms/base_requestor.rb

```diff
@@ -53,17 +55,19 @@ def soap_envelope(namespaces)
     def soap_header(xml)
       xml.__send__('ns1:Security') do
         xml.__send__('ns1:UsernameToken') do
-          xml.__send__('ns1:Username', ENV['SOAP_CLIENT_USERNAME'])
-          xml.__send__('ns1:Password', 'Type' => ENV['SOAP_CLIENT_PASSWORD_TYPE']) { xml.text ENV['SOAP_CLIENT_PASSWORD'] }
+          xml.__send__('ns1:Username', Rails.configuration.x.ccms_soa.client_username)
+          xml.__send__('ns1:Password', 'Type' => Rails.configuration.x.ccms_soa.client_password_type) do
+            xml.text(Rails.configuration.x.ccms_soa.client_password)
+          end
         end
       end
     end
 
     def ns3_header_rq(xml)
       xml.__send__('ns3:TransactionRequestID', transaction_request_id)
       xml.__send__('ns3:Language', 'ENG')
-      xml.__send__('ns3:UserLoginID', ENV['USER_LOGIN'])
-      xml.__send__('ns3:UserRole', ENV['USER_ROLE'])
+      xml.__send__('ns3:UserLoginID', Rails.configuration.x.ccms_soa.user_login)
+      xml.__send__('ns3:UserRole', Rails.configuration.x.ccms_soa.user_role)
```

Comment:
if you add a `config` method in `BaseRequestor `
```ruby
def config
  Rails.configuration.x.ccms_soa
end
```

could you simplify all the uses of `Rails.configuration.x.ccms_soa....` to `config...` ?

Reply:
Nice suggestion, doing it!